content/documentation-v1/getting-started.md

```diff
@@ -157,43 +145,45 @@ The names of the files will effect the final URLs of your publication. By defaul
 {{< /q-class >}}
 
-For the ordering of the pages, in the example above we’ve listed the files and directories as they would appear in the publication’s table of contents. When looking in the actual `content` directory on your computer or in your text editor, however, they will almost certainly not appear in the proper publication order. More likely, they’ll appear alphabetically or by date modified, which is also how Quire will order them when building and previewing your publication. You can adjust this by [assigning a `weight` to each page](/documentation/pages/#organize-pages-in-the-right-order) in its page YAML.
+ When looking in the `content` directory on your computer or in your text editor, project files usually either appear alphabetically or by date modified, which is also how Quire will order them when building and previewing your publication. You can adjust this by adding `order` value in page YAML of each Markdown file. Learn more in [*Page Types & Structure*](/documentation/pages/#organize-pages-in-the-right-order).
 
 There are some other important rules and tips to keep in mind:
 
-1. **To create a new file, select "New File" in the text editor menu.** You can also right click or press control click on a folder and select "New File".
+1. **To create a new file, right-click on the `content` directory and select "New File".**
 
-2. **File names should be lowercase, with no spaces. Always include the .md suffix.** If file names contain more than one word, use a hyphen to separate them. Make sure that the file name includes either a `.md` or `.yml` extension.
+2. **File names should be lowercase**
 
-3. **Don't use `index.md` or `_index.md` files.** Though common for users with previous static site or web development experience, you should not use `index.md` or `_index.md` files in your Quire project. Because of the way {{< q-def "Hugo" >}} is modeled, these work against the linear ordering of the publication and break the *Next* and *Previous* page navigation in Quire.
+3. **Use hyphen in place of spaces**
 
-## Prepare Images and Text
+4. **Always include the .md suffix.**
 
-TK
+## Add Text and Images
 
-## Preview and Edit a Project
+Once an outline has been created, text and images can be added to the project.
 
-Quire lets you preview the current version of your site in a web browser, and will update the preview as you edit the files.
+In many cases, text will originate as a Word document. You can use Pandoc to convert Word documents into Markdown. Instructions can be found in the [YAML & Markdown](/documentation/fundamentals/#microsoft-word-to-markdown-conversion) section of this guide.
 
-**To run the preview:**
+Once images are added to the `_assets/images` folder and entries have been added to the `figures.yaml`, then you can use shortcodes to insert the images into their respective Markdown files. More information can be found in the [Figure Images](/documentation/figure-images/) section of this guide.
```

Comment:
I'm not sure if we're consistent, but my instinct is to always follow directory names with a slash: `_assets/images/`. The slash is the equivalent to a file suffix, like `.md`. What do you think?

Reply:
Noted! I've fixed it here but will likely need to correct it elsewhere in this guide and in the Getty Quire Guide.